package-lock.json

```diff
@@ -7878,4 +7878,4 @@
       }
     }
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
add new line -> https://unix.stackexchange.com/questions/484802/should-the-last-line-in-a-configuration-file-have-a-newline-after-it